src/modules/endpoint/sagas/loadEvents.js

```diff
@@ -25,7 +26,10 @@ function * loadEvents (
 
   console.debug('Fetching events')
 
-  const payload = yield call(() => createEventsEndpoint(baseUrl).request({ city, language }))
+  const appSettings = new AppSettings()
+  const apiUrlOverride = yield call(appSettings.loadApiUrlOverride)
+
+  const payload = yield call(() => createEventsEndpoint(apiUrlOverride || baseUrl).request({ city, language }))
```

Comment:
Those two cases `apiUrlOverride` is null or not should be tested in `loadEvents.spec.js`.